graphql/resolvers.go

```diff
@@ -273,6 +273,17 @@ func (r *taskResolver) ReliesOn(ctx context.Context, at *restModel.APITask) ([]*
 	return dependencies, nil
 }
 
+func (r *projectResolver) IsFavorite(ctx context.Context, at *restModel.APIProjectRef) (bool, error) {
+	p, err := model.FindOneProjectRef(*at.Identifier)
+	if err != nil || p == nil {
```

Comment:
I think we should propagate the error to the client easier debugging (logs to bugsnag)
also it's possible that err could be a mongodb error which should be returned with `InternalServerError`